tests/test_parser.py

```diff
@@ -1829,6 +1829,20 @@ def test_prioritization(self):
             res = l.parse("a")
             self.assertEqual(res.children[0].data, 'a')
 
+            grammar = """
+            start: a | b
+            a.2: "A"+
+            b.1: "A"+ "B"?
+            """
+
+            l = _Lark(grammar)
+            res = l.parse("AAAA")
+            self.assertEqual(res.children[0].data, 'a')
+
+            l = _Lark(grammar, priority="invert")
+            res = l.parse("AAAA")
+            self.assertEqual(res.children[0].data, 'b')
+
```

Comment:
I imagined here to also test that the input `AAB` gets correctly identified and doesn't cause problems. But invert is also nice to test.